### PR TITLE
SRILoadcell_ADC Integration

### PR DESCRIPTION
**Pull request by jkotar3**
## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [x] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [x ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/imsenthur/opensourceleg/blob/master/CODE_OF_CONDUCT.md) document.
- [x ] I've read the [`CONTRIBUTING.md`](https://github.com/imsenthur/opensourceleg/blob/master/CONTRIBUTING.md) guide.
- [ x] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [x ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: opensourceleg/sensors/loadcell.py [hunk 1]
```diff
@@ -6,29 +6,22 @@
...
     def __init__(
         self,
         matrix=None,
         excitation_voltage=5,
-        adc: ADCBase = None,
-        adc_board="neurobionics",
         **kwargs,
     ):
-        """Initializes Sunrise Instruments Loadcell"""
+        """Initializes Sunrise Instruments Loadcell using ADC"""
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:20]: be specific about ADC

File: opensourceleg/sensors/loadcell.py [hunk 2]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
 
     @property
     def adc(self) -> ADCBase:
+        """Returns the ADS131M0x object of the Loadcell"""
         return self._adc
 
     @property
     def is_streaming(self) -> bool:
-        return self.adc._streaming
+        """Returns whether the ADC of the Loadcell is streaming data or not"""
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:57]: Returns True if ADC is streaming data

File: opensourceleg/sensors/loadcell.py [hunk 3]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
+        return (bool)(self.adc.is_streaming)
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:58]: Casting?

File: opensourceleg/sensors/loadcell.py [hunk 4]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def fx(self) -> float:
-        return self.data[0]
+        """Returns the force in the x direction of the loadcell"""
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:62]: Mention force & torque units for all properties

File: opensourceleg/sensors/loadcell.py [hunk 5]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
+        return (float)(self.data[0])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:63]: casting

File: opensourceleg/sensors/loadcell.py [hunk 6]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def fy(self) -> float:
-        return self.data[1]
+        """Returns the force in the y direction of the loadcell"""
+        return (float)(self.data[1])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:68]: casting

File: opensourceleg/sensors/loadcell.py [hunk 7]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def fz(self) -> float:
-        return self.data[2]
+        """Returns the force in the z direction of the loadcell"""
+        return (float)(self.data[2])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:73]: casting

File: opensourceleg/sensors/loadcell.py [hunk 8]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def mx(self) -> float:
-        return self.data[3]
+        """Returns the moment in the x direction of the loadcell"""
+        return (float)(self.data[3])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:78]: casting

File: opensourceleg/sensors/loadcell.py [hunk 9]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def my(self) -> float:
-        return self.data[4]
+        """Returns the moment in the y direction of the loadcell"""
+        return (float)(self.data[4])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:83]: casting

File: opensourceleg/sensors/loadcell.py [hunk 10]
```diff
@@ -56,32 +49,40 @@ def update(self) -> None:
...
 
     @property
     def mz(self) -> float:
-        return self.data[5]
+        """Returns the moment in the z direction of the loadcell"""
+        return (float)(self.data[5])
```
Comment (JapmanGill) [inline opensourceleg/sensors/loadcell.py:88]: casting

File: opensourceleg/sensors/loadcell.py [hunk 11]
```diff
@@ -6,29 +6,22 @@
 
 import numpy as np
 from adc import ADS131M0x
-from base import LoadcellBase, ADCBase
+from base import ADCBase, LoadcellBase
 
 
-class SRILoadcell(LoadcellBase):
+class SRILoadcell_ADC(LoadcellBase):
```
[the remark this answers is not in the document]
Comment (JapmanGill) [reply opensourceleg/sensors/loadcell.py:12]: Add the old loadcell class, and rename it to SRILoadcell_Legacy

**Review by JapmanGill — DISMISSED** [review]

**Comment by senthurayyappan** [comment]
@all-contributors add @jkotar3 as a contributor for tests, bug reports, and code

**Comment by senthurayyappan** [comment]
@all-contributors add @JapmanGill as a contributor for tests, bug reports, and code

**Pull request merged**